### PR TITLE
Add consent page for the Security Advisor SHIELD study.

### PR DESCRIPTION
**Pull request by Osmose**
This is mostly a copy of #3271, but with language changed for the new study.

We're hoping to get this deployed in time for releasing the study on Thursday. :D

See [bug 1295216](https://bugzilla.mozilla.org/show_bug.cgi?id=1295216) for tracking for this study.

### CODE REVIEW

**Comment by muffinresearch** [comment]
Is there a reason for the `src/olympia/files/fixtures/files/jétpack.xpi` being part of this diff?

File: src/olympia/pages/templates/pages/shield_study_6.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<p>This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.</p>
+<h3 id="this-particular-study">This Particular Study</h3>
+<p>In addition to the data collected by all Shield Studies and your voluntary answers to the followup survey, here are the <strong>key things you should know</strong> about what is happening when you participate in this study:</p>
+<ul>
+<li>We collect basic usage data regarding your interaction with the experimental add-on, such as data when you use a new security feature.</li>
+<li>We collect information about your visits to a few popular sites and if you have a saved login for that site. No login credentials will be tracked or saved.</li>
+<li>As you would expect, we do not collect any information when you are in private browsing mode.</li>
+</ul>
+<p>You can learn more about the data collection for this study <a href="https://github.com/mozilla/security-advisor-shield-study/blob/master/domains.txt">here</a></p>
```
Comment (muffinresearch) [inline src/olympia/pages/templates/pages/shield_study_6.html]: Would be really nice to fix this link to say more than "here" for better accessibility.

**Comment by muffinresearch** [comment]
@Osmose can you fix up the couple of nits and then we'll get this landed so it will make the tag. The tag happens at 5pm UTC today (Tuesday)

**Comment by EnTeQuAk** [comment]
Oops, I didn't see this pull request. I already merged https://github.com/mozilla/addons-server/pull/3950 which is why you're getting merge conflicts now :(

Sorry about that!

**Comment by Osmose** [comment]
> Is there a reason for the src/olympia/files/fixtures/files/jétpack.xpi being part of this diff?

Nope, just an artifact of weird unicode issues my filesystem was having with my checkout.

**Comment by Osmose** [comment]
Rebased and amended! I amended instead of adding another commit to avoid having that odd unicode thing in the git history at all, but the only two changes were removing it and expanding the link as requested.

**Comment by EnTeQuAk** [comment]
you can see how the page you landed looks like in our development environment: https://addons-dev.allizom.org/en-US/firefox/shield_study_6

This will go live on Thursday.

**Comment by muffinresearch** [comment]
It's now on stage here: https://addons.allizom.org/en-US/firefox/shield_study_6

**Pull request merged**